From: uazo <uazo@users.noreply.github.com>
Date: Tue, 24 Feb 2026 10:14:12 +0000
Subject: Enable Socket Pool Limit Randomization

Anticipates the activation of Socket Pool Limit Randomization scheduled for v147
see https://chromium-review.googlesource.com/c/chromium/src/+/7515640
---
 net/base/features.cc | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/net/base/features.cc b/net/base/features.cc
--- a/net/base/features.cc
+++ b/net/base/features.cc
@@ -563,31 +563,31 @@ BASE_FEATURE(kVerifyMTCs, base::FEATURE_DISABLED_BY_DEFAULT);
 #endif
 
 BASE_FEATURE(kTcpSocketPoolLimitRandomization,
-             base::FEATURE_DISABLED_BY_DEFAULT);
+             base::FEATURE_ENABLED_BY_DEFAULT);
 
 BASE_FEATURE_PARAM(double,
                    kTcpSocketPoolLimitRandomizationBase,
                    &kTcpSocketPoolLimitRandomization,
                    "TcpSocketPoolLimitRandomizationBase",
-                   0.0);
+                   0.000001);
 
 BASE_FEATURE_PARAM(int,
                    kTcpSocketPoolLimitRandomizationCapacity,
                    &kTcpSocketPoolLimitRandomization,
                    "TcpSocketPoolLimitRandomizationCapacity",
-                   0);
+                   256);
 
 BASE_FEATURE_PARAM(double,
                    kTcpSocketPoolLimitRandomizationMinimum,
                    &kTcpSocketPoolLimitRandomization,
                    "TcpSocketPoolLimitRandomizationMinimum",
-                   0.0);
+                   0.01);
 
 BASE_FEATURE_PARAM(double,
                    kTcpSocketPoolLimitRandomizationNoise,
                    &kTcpSocketPoolLimitRandomization,
                    "TcpSocketPoolLimitRandomizationNoise",
-                   0.0);
+                   0.2);
 
 BASE_FEATURE(kNetTaskScheduler, base::FEATURE_ENABLED_BY_DEFAULT);
 BASE_FEATURE_PARAM(bool,
--
